mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -416,3 +418,23 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 a
 		ARG 2 b
 		ARG 4 c
+	METHOD method_41303 bezierInterpolate (FFFFF)F
+		COMMENT An interpolation for a point on a cubic bezier curve. This is done
+		COMMENT on each vector component independently.
+		COMMENT
+		COMMENT <p>Given endpoints {@code p0} and {@code p3}, {@code 3 * (p1 - p0)} as
+		COMMENT derivative at {@code p0}, and {@code 3 * (p3 - p2)} as derivative at
+		COMMENT {@code p3}, these are sufficient to define a unique cubic bezier curve.
+		COMMENT
+		COMMENT @param delta the progress from {@code p0} to {@code p3}
```

Comment:
These are causing lint errors (add to param javadoc maybe?)

Reply:
right, haven't used yarn in a while and forgot about that